scripts/winesapos_convert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Script by @GuestSneezeOSDev
+echo "Convering system to winesapOS in 3"
+sleep 1
+echo "Convering system to winesapOS in 2"
+sleep 1
+echo "Convering system to winesapOS in 1"
+sleep 1
+echo "System is converting ..."
+pacman -Sy --noconfirm
+useradd -m winesap
+echo "winesap:winesap" | chpasswd
+pacman -S git wget flatpak base-devel --noconfirm
+wget https://winesapos.lukeshort.cloud/repo/iso/winesapos-4.1.0/_test/winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
+pacman -U --noconfirm winesapos-4.1.0-beta.1-minimal-rootfs.tar.zst
+rm *.zst
+touch /etc/os-release-winesapos
+echo 'NAME="winesapOS"' >> /etc/os-release-winesapos
+echo 'PRETTY_NAME="winesapOS"' >> /etc/os-release-winesapos
+echo 'ID=winesapOS' >> /etc/os-release-winesapos
+echo 'ID_LIKE=arch' >> /etc/os-release-winesapos
+echo 'VERSION_ID=4.1.0-beta.3' >> /etc/os-release-winesapos
+echo 'HOME_URL="https://github.com/LukeShortCloud/winesapOS"' >> /etc/os-release-winesapos
+echo 'SUPPORT_URL="https://github.com/LukeShortCloud/winesapOS/issues"' >> /etc/os-release-winesapos
+echo 'BUG_REPORT_URL="https://github.com/LukeShortCloud/winesapOS/issues"' >> /etc/os-release-winesapos
+echo "[jupiter-staging]" >> /etc/pacman.conf
+echo "Server = https://steamdeck-packages.steamos.cloud/archlinux-mirror/\$repo/os/\$arch" >> /etc/pacman.conf
+echo "SigLevel = Never" >> /etc/pacman.conf
+echo "[winesapos]" >> /etc/pacman.conf
+echo "Server = https://winesapos.lukeshort.cloud/repo/winesapos/\$arch/" >> /etc/pacman.conf
+echo "SigLevel = Never" >> /etc/pacman.conf
+git clone https://aur.archlinux.org/yay.git
+cd yay
+makepkg -si --noconfirm
+cd ..
+rm -rf yay
+yay -S --noconfirm waydroid waydroid-biglinux waydroid-image waydroid-image-gapps waydroid-image-halium waydroid-magisk waydroid-openrc google-chrome protonup-qt heroic-games-launcher heroic-games-launcher-bin prismlauncher prismlauncher-bin lutris-wine-meta
```

Comment:
In addition to `yay`, you may want to use `pacman` to install non-AUR packages. You can quickly see all of the packages winesapOS installs by running this command  in the git repository:

```
grep pacman_install scripts/winesapos-install.sh
```